### DISCUSSION TITLE
Client/server config for Mobs playing music with backpacks

### DISCUSSION THREAD

**Discussion by TonyD6** [P3pp3rF1y/SophisticatedBackpacks#1154] [Ideas]
Currently you can only disable mobs playing jukebox backpacks in the per world config. this really sucks for pack developers

**Comment by P3pp3rF1y** [comment]
Why is that? There's defaultconfigs folder or does that not help?

**Reply by TonyD6** [reply]
im not sure what your asking. Sohpisticated backpacks doesnt have anything located in the defaultconfigs folder

**Comment by cloodstool** [comment]
I have a config file in the config folder for my server, but I don't see anything pertaining to mobs spawning with backpacks
